README.md

```diff
@@ -17,11 +17,36 @@ a.process
 
 ## Unit tests ##
 
-To be sure that unit tests run properly, you can run them in a docker container.
+To be sure that unit tests run properly, you can run them in a Docker container.
 
 ```bash
 docker run -ti --rm -v $(pwd):/opt/aliquot ruby:2.7.4 bash
 cd /opt/aliquot
 bundle install
 bundle exec rspec
+exit
```

Comment:
Feels a little superfluous :smile:

Reply:
```suggestion
```

Reply:
Keep in mind that these instructions are written inside the container, so `exit` is to put you back into your local shell. Kind of superfluous, but also something you'll inevitably be writing afterwards (unless you close the window or CTRL+D).

Reply:
Your sentence can be read both as an argument for keeping it and for removing it :slightly_smiling_face: 

It's not important to me whether or not we keep it.